.github/workflows/examples.yml

```diff
@@ -16,13 +16,24 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: setup-python${{ matrix.python-version }}
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install
+    - name: Setup cache
+      uses: actions/cache@v2
+      env:
+        cache-name: daily-example
+      with:
+        path: /opt/**/site-packages
```

Comment:
I'm not quite sure that this path specification is cool or not at all.

Ref: https://github.com/actions/setup-python/#hosted-tool-cache

Reply:
Maybe safer to get it from the runtime? 

```bash
python -c "import site; print(site.getsitepackages())"
```